testnet/testnet.go

```diff
@@ -219,7 +223,7 @@ func (tn *TestNet) AddDetails(dd db.DeploymentDetails) error {
 		}
 	}
 	for i := len(tn.CombinedDetails.Resources); i < oldCD.Nodes; i++ {
-		tn.CombinedDetails.Resources = append(tn.CombinedDetails.Resources, tn.GetNodeResources(i))
+		tn.CombinedDetails.Resources = append(tn.CombinedDetails.Resources, tn.GetNodeResources(tn.Nodes[i]))
```

Comment:
Is there a reason for that change?

Reply:
Yeah, i changed the interface so that it can extract information from arbitrary nodes, rather than having to add them first